manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.yaml

```
@@ -0,0 +1,8 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 9.30.2

```

```suggestion
PackageVersion: 9.30
```
This needs to match control panel, or there will be an upgrade loop.